tensorflow/python/ops/ragged/ragged_math_ops.py

```diff
@@ -630,6 +637,18 @@ def reduce_mean(input_tensor, axis=None, keepdims=None, name=None):
       return total / count
 
 
+def reduce_variance(input_tensor, axis=None, keepdims=False, name=None):
+  """For docs, see: _RAGGED_REDUCE_DOCSTRING."""
+  with ops.name_scope(name, 'RaggedReduceVariance', [input_tensor, axis]):
+    square_of_input = gen_math_ops.square(input_tensor)
```

Comment:
Using `gen_math_ops.square` works here, but it's preferable to just use `math_ops.square`.  (`gen_math_ops.py` is the generated python stub file, and all of its symbols are imported into `math_ops.py`.)

Reply:
done the change